test/extensions/filters/network/dubbo_proxy/route_matcher_test.cc

```diff
@@ -914,6 +914,298 @@ name: config
   EXPECT_THROW(SingleRouteMatcherImpl m(config, context), EnvoyException);
 }
 
+TEST(DubboRouteMatcherTest, RouteActionMetadataMatch) {
+  const std::string yaml = R"EOF(
+name: config
+interface: org.apache.dubbo.demo.DemoService
+routes:
+  - match:
+      method:
+        name:
+          exact: "method1"
+    route:
+      cluster: cluster1
+      metadata_match:
+        filter_metadata:
+          envoy.lb:
+            k1: v1
+            k2: v2
+  - match:
+      method:
+        name:
+          exact: "method2"
+    route:
+      cluster: cluster2
+)EOF";
+
+  envoy::extensions::filters::network::dubbo_proxy::v3::RouteConfiguration config =
+      parseRouteConfigurationFromV2Yaml(yaml);
+  auto invo = std::make_shared<RpcInvocationImpl>();
+
+  // Match with metadata
```

Comment:
nit: it would be better to make the comment end wich `.`.

Reply:
Done